### ISSUE TITLE
Add recipe for rendering Markdown and MDX content in an RSS feed

### ISSUE THREAD

**Issue by bholmesdev** [withastro/docs#10317]
### 📚 Subject area/topic

RSS, MDX

### 📋 Suggested page

https://docs.astro.build/en/recipes/

### 📋 General description or bullet points (if proposing new content)

Our [MDX compiledContent discussion](https://github.com/withastro/roadmap/discussions/419#discussioncomment-11488764) has been open for two years now. Users were mainly interested in this feature to render rich text content in their RSS feeds.

We discovered that the container API can finally address this use case. Chris did some legwork [building an example with the container API](https://github.com/withastro/roadmap/discussions/419#discussioncomment-11256751), but it's fairly complex and difficult for new users to discover. We also got [a recent follow-up](https://github.com/withastro/roadmap/discussions/419#discussioncomment-11488764) that outlines an alternative solution due to issues in Chris' example.

I think this use case is common enough that we should create a recipe for it. Find the simplest option using today's APIs, and log any roadmap items to improve the container API if we find it necessary while writing the docs.

**Comment by sarah11918** [comment]
Would welcome a recipe, which (reminder) would need a proper structure around it to *accomplish a specific task outlined in the recipe*. Things like:

- state a problem to solve/specific goal to accomplish (e.g.  "Generating full-text RSS feeds from remote Markdown/MDX" not "general theory", explanation of how things work focusing on the end goal the user has (RSS feeds when the content is remote) vs the intermediate goal "rendering remote Markdown" because they might not be thinking of searching for one step in the process)
- write an opinionated step-by-step example that achieves that goal, which you can do because you state prerequisites/assumptions and write something that someone can follow and it will actually work
- show how to check that implementing it was successful

If someone would like to turn this into a recipe, we would happily take a PR! And I think it could also be helpful to include a "Community Resource" link to [delucis/astro-blog-full-text-rss](https://github.com/delucis/astro-blog-full-text-rss/)!

**Comment by sarah11918** [comment]
Discussed on Talking and Doc'ing that it feels super difficult to create a recipe someone can follow and it's much preferable if someone would build an integration or functionality to handle all the transformation necessary, then we can recommend this package in Docs!

**Comment by bholmesdev** [comment]
Makes sense @sarah11918! How are we tracking that? Should we make a roadmap discussion for an integration?
